core/src/main/java/org/apache/accumulo/core/fate/user/FateMutatorImpl.java

```diff
@@ -79,6 +82,33 @@ public FateMutator<T> putCreateTime(long ctime) {
     return this;
   }
 
+  @Override
+  public FateMutator<T> putReservedTx(FateStore.FateReservation reservation) {
+    Condition condition = new Condition(TxColumnFamily.RESERVATION_COLUMN.getColumnFamily(),
+        TxColumnFamily.RESERVATION_COLUMN.getColumnQualifier()).setValue(NOT_RESERVED);
+    mutation.addCondition(condition);
+    TxColumnFamily.RESERVATION_COLUMN.put(mutation, new Value(reservation.toString()));
```

Comment:
Its nice to avoid calling toString() when its return value is used with serialized data as the expectations around toString() are not well defined. For example someone could change the toString impl to improve a log message or something like that and not realize the implications for serialized data.

```suggestion
    TxColumnFamily.RESERVATION_COLUMN.put(mutation, new Value(reservation.getSerialized()));
```

Reply:
Yes, good call. I now only use the serialized value